test/e2e/playbook/provision.yaml

```diff
@@ -6,6 +6,12 @@
     - cri_runtime: "{{ cri_runtime }}"
     - is_containerd: false
     - is_crio: false
+    - containerd_release: "{{ containerd_release }}"
+    - is_containerd_src: false
```

Comment:
If we would like to use build from source, how can we set the variable value to true? My point is, the value is hardcoded here and the only way to set it to true is my modifying the playbook right?

Reply:
`is_{containerd,crio}_src` are bogus/unused variables which have been removed by a subsequent force-push.

To use a self-compiled runtime you should do a `containerd_src="path-to-containerd-directory-with-precompiled-sources" {run_test.sh|runs.sh} path-to-testcase` and the scripts/playbook should figure out the rest by themselves.